netlify/landings/prism.yaml

```
@@ -0,0 +1,79 @@
+path: /prism
+title: Open Source Mock Server
+color: purple-darker
+relatedTags:
+  - blog-mocking
+hero:
+  subtitle: Build APIs and accelerate API development with instant mock servers
+  title: Write the spec. Prism will do the rest.
+featureSection:
+  actionBar:
+    buttons:
+      - href: 'https://github.com/stoplightio/prism'
+        title: Try Prism now
+    enabled: true
+    text: 'Want to mock your API?'
```

```suggestion
    text: Want to create a mock API?
```

Mock your API makes it sound like creating a fake impl of an existing API, which is a valid use case but not the primary one, and not the first thing that people think of when they think about mocking: parallelizing development efforts by getting feedback before they waste time writing code.